R/classes-plate.R

```diff
@@ -202,19 +208,7 @@ Plate <- R6::R6Class(
     #' By default `avg`. For now it is the only available method
     #' @param inplace Whether the method should produce new plate with adjusted
     #' values or not, By default `TRUE` - operates on the current plate.
-    blank_adjustment = function(method = "avg", in_place = "TRUE") {},
-
-    #' @description
-    #' Function takes the data 1D array and returns the data in the form of
-    #' a 2D array, that represents the spatial arrangement of the samples
-    #' on the plate, ie. 12x8 grid.
-    #'
-    #' @param data 1D array of data represents for example counts, dilutions
-    #' or sample types
-    #'
-    #' @return A 2D array representing data with the spatial arrangement
-    #' of the samples
-    layout = function(data) {}
```

Comment:
why are we removing this function? I personaly won't be using it to plot plate but was there any reason why we come  up with this function?

Reply:
I figured that it will be better to store layout information simply in a format of matrix with named rows and columns. So  that we can index it much faster